docs/release-notes/release-notes-0.16.1.md

```diff
@@ -0,0 +1,10 @@
+# Release Notes
+
+## Bug Fixes
+
+* [Fix a bug where lnd crashes when psbt data is not fully 
```

Comment:
This PR should be against the `0-16-1-staging` branch, which already have this file